lua/mapvote/client/modules/thumb_downloader.lua

```diff
@@ -31,7 +77,9 @@ function ThumbDownloader:DownloadAll()
     local mapsToDownload = {}
     for _, map in pairs( self.mapsToDownload ) do
         local wsid = self.workshopIDLookup[map]
-        if wsid then
+        if self.urlOverrides[map] then
+            self:DownloadThumbnail( map, self.urlOverrides[map] )
```

Comment:
Might be nice if there was a mechanism for falling back to the real Icon if a custom icon fails.

With the way it's set up right now, I don't have any good suggestions for where to make those changes.

Reply:
if you are already making them download a backup icon for the map on join then you probably wouldnt even need the http icon. The point of this was just to make it really easy to add map icons without adding any download time on join.